profile/src/v100/networks/network/profile_network.rs

```diff
@@ -40,6 +47,7 @@ impl ProfileNetwork {
         Self {
```

Comment:
Assert that all personas are on the same network and on the same network as accounts